### DISCUSSION TITLE
Referring GraphQL schema files from a Maven dependency jar

### DISCUSSION THREAD

**Discussion by MovingToContainers** [Netflix/dgs-framework#1511] [Q&A] [answered]
Hi team, 

is there any way to refer gql schema file from maven dependency jar.

**Comment by MovingToContainers** [comment]
can any one help me with the syntax to refer the schema file from the Maven dependency jar.

dgs:
  graphql:
    schema-locations:

**Comment by srinivasankavitha** [comment, marked as answer]
We don't support  configuring this in the framework.

**Reply by MovingToContainers** [reply]
Hi @srinivasankavitha , Thank you for the swift response. could you let me know, Is there any way to refer the schema file from any external location using Spring cloud config server?

**Reply by MovingToContainers** [reply]
Thank you..!

**Comment by srinivasankavitha** [comment]
I'm not familiar enough with that unfortunately to make a recommendation.
